WordPress/Classes/ViewRelated/Stats/Traffic/StatsTrafficDatePickerView.swift

```diff
@@ -16,9 +18,11 @@ struct StatsTrafficDatePickerView: View {
                 Text(viewModel.period.label)
                     .style(TextStyle.bodySmall(.emphasized))
                     .foregroundColor(Color.DS.Foreground.primary)
+                    .dynamicTypeSize(...maxDynamicTypeSize)
                 Image(systemName: "chevron.down")
                     .font(.system(size: 8))
```

Comment:
I noticed the down chevron wasn't responding to changing the font size, and then it used a fixed font size. I tried setting it to `caption2` since it's smaller than the other chevrons in the designs and I think it looks good:

| Default font size | Larger font size |
| - | - |
| <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1898325/9c1dc4c0-e845-4c29-a9d0-f80fcaddb273" /> | <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1898325/aa6e4261-19c7-43bf-9a35-c397e96262ae" /> |

Wdyt?

Reply:
Yes, I agree!